src/index.js

```diff
@@ -36,6 +36,14 @@ const brandColor = /** @type {const} */ ([174, 128, 255]);
 						}
 					},
 				}),
+			type: () => prompts.select({
```

Comment:
I might go with `appType` or similar, just to be a bit more specific as to what it refers to